__tests__/app.test.js

```diff
@@ -178,4 +178,37 @@ describe.only("GET /api/articles/:article_id/comments", ()=>{
       expect(msg).toBe('No Comments')
     })
   })
+})
+describe("POST /api/articles/:article_id/comments", ()=>{
+  test("201: Add a comment for given article_id", ()=>{
+    const testBody = {username: 'icellusedkars', body: 'Primus sucks'}
+    return request(app)
+    .post('/api/articles/1/comments')
+    .send(testBody)
+    .expect(201)
+    .then(({text})=>{
+      expect(text).toBe('Primus sucks')
+    })
+  })
+
+  test("404: returns 'Not Found' if article_id doesn't exist", ()=>{
+    const testBody = {username: 'icellusedkars', body: 'Primus sucks'}
+    return request(app)
+    .post('/api/articles/1337/comments')
+    .send(testBody)
+    .expect(404)
+    .then(({text})=>{
+      expect(text).toBe('Not Found')
+    })
+  })
+  test("404: returns 'Not Found' if author doesn't exist", ()=>{
+    const testBody = {username: 'deceptacon', body: 'Who took the ram from the ramalamadingdong?'}
+    return request(app)
+    .post('/api/articles/1/comments')
+    .send(testBody)
+    .expect(404)
+    .then(({text})=>{
+      expect(text).toBe('Not Found')
+    })
+  })
```

Comment:
There are a few missing tests here. What about if the article id is invalid, a missing comment, a missing user, an empty body??